contributors/chairs-and-techleads/leadership-changes.md

```diff
@@ -0,0 +1,40 @@
+# Leadership Changes
+
+This document covers steps needed to propose changes to SIG/WG/UG leadership.
+
+- [ ] Discuss the proposed changes with the current leadership.
+
+- [ ] Send an email to the SIG/WG/UG mailing list and cc the [kubernetes-dev]
+mailing list. At a minimum, the email should contain:
+  - [ ] intent to step down as the current lead
+  - [ ] if nominating another lead
+    - [ ] 1-2 lines about why they are being nominated
+    - [ ] contacts to privately reach out to for questions (current leads)
+          or concerns (current leads + [steering-private]) about the nomination
+  - [ ] if this was discussed in a meeting, link to meeting notes
+  - [ ] lazy consensus deadline of at least one week
+
+- [ ] If nominating another lead, ensure that they
+  - [ ] are a Kubernetes GitHub org [member]
+  - [ ] have completed the [Inclusive Leadership Training]
+
+- [ ] Once lazy consensus has been achieved, updating the following
+      files in the respective repos:
```

Comment:
My SIG was unable to reach lazy consensus because multiple leads ran for an open position, so we had an election using CIVS with voters defined from the SIG membership defined in kubernetes/org. Should we include some possible options for when lazy consensus isn't achieved?

Reply:
> Should we include some possible options for when lazy consensus isn't achieved?

That's a good point!

@mrbobbytables do you know what process was followed for the SIG lead election in this case? I know contribex was involved, but I can't recall the details.

Reply:
ContribEx was consulted, but was not involved in the execution IIRC. We do have an oustanding item to document the process. - TL;DR -  ContribEx will handle the execution for a CIVS election if provided a list of emails and candidates as a neutral party.

Reply:
Since https://github.com/kubernetes/community/issues/5096 is in flight I'm hoping we can avoid direct references/codifying CIVS -- maybe we can keep this general and just indicate that if there isn't lazy consensus,

- an election should be held
- the eligible voters will be the members of the SIG as defined in kubernetes/org
- contribex should be contacted to assist in the administration of the election

Reply:
Thanks @mrbobbytables and @ehashman!

Added a note about elections. We can add detailed steps/checklist for the election once we figure out the process: https://github.com/kubernetes/community/issues/4586